src/Data/Fin/Properties.agda

```diff
@@ -375,10 +387,11 @@ toℕ-inject {i = suc i} (suc j) = cong suc (toℕ-inject j)
 -- inject+
 ------------------------------------------------------------------------
 
-toℕ-inject+ : ∀ {m} n (i : Fin m) → toℕ i ≡ toℕ (inject+ n i)
+toℕ-inject+ : ∀ {m} n (i : Fin m) → toℕ i ≡ toℕ (i ↑ˡ n)
+{-
```

Comment:
Still 3 instances of commented out code and types in this file as well.

Reply:
grrr. sorry. too busy attending to adding to CHANGELOG.md, and not enough to the deletion part of the deal...